hips/hip-9999.md

```diff
@@ -0,0 +1,321 @@
+---
+hip: 9999
+title: "Enhance Helm Pull Request Lifecycle Policy"
+authors: ["Scott Rigby <scott@r6by.com>"]
+created: "2025-11-05"
+type: "process"
+status: "draft"
+---
+
+## Abstract
+
+This proposal enhances the Helm pull request lifecycle policy to improve organization and automation of PR triaging, review, changelog generation, and documentation. The proposal introduces a standardized system based on Conventional Commits syntax for PR titles and corresponding labels, establishing clear criteria for review-readiness and merge-readiness while automating release note generation.
```

Comment:
* Provide a link to Helm existing pull request lifecycle policy
* Add a reference to Conventional Commits